src/config/config.js

```diff
@@ -219,6 +219,12 @@ export const config = convict({
         format: Number,
         default: fourHoursMs,
         env: 'SESSION_CACHE_TTL'
+      },
+      apiEndpoint: {
+        doc: 'Grants UI Backend API endpoint',
+        format: String,
```

Comment:
Could this be 'url'  to use Convicts validation?

Reply:
Do you want to install convict-format-with-validator? It's an old package... Could do our own validator?